idaes/models/properties/examples/tests/test_methanol_ideal_VLE.py

```diff
@@ -0,0 +1,298 @@
+#################################################################################
+# The Institute for the Design of Advanced Energy Systems Integrated Platform
+# Framework (IDAES IP) was produced under the DOE Institute for the
+# Design of Advanced Energy Systems (IDAES), and is copyright (c) 2018-2021
+# by the software owners: The Regents of the University of California, through
+# Lawrence Berkeley National Laboratory,  National Technology & Engineering
+# Solutions of Sandia, LLC, Carnegie Mellon University, West Virginia University
+# Research Corporation, et al.  All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and
+# license information.
+#################################################################################
+"""
+Author: Brandon Paul
+"""
+
+import pytest
+from pyomo.environ import (
+    check_optimal_termination,
+    ConcreteModel,
+    Set,
+    value,
+    Var,
+    units as pyunits,
+)
+from pyomo.util.check_units import assert_units_consistent
+from pyomo.common.unittest import assertStructuredAlmostEqual
+
+from idaes.core import LiquidPhase, VaporPhase, Component
+from idaes.core.util.model_statistics import (
+    degrees_of_freedom,
+    fixed_variables_set,
+    activated_constraints_set,
+)
+from idaes.core.solvers import get_solver
+
+from idaes.models.properties.modular_properties.base.generic_property import (
+    GenericParameterBlock,
+)
+
+from idaes.models.properties.modular_properties.state_definitions import FPhx
+from idaes.models.properties.modular_properties.eos.ideal import Ideal
+from idaes.models.properties.modular_properties.phase_equil import SmoothVLE
+from idaes.models.properties.modular_properties.phase_equil.bubble_dew import \
+        IdealBubbleDew
+from idaes.models.properties.modular_properties.phase_equil.forms import \
+    fugacity
+
+from idaes.models.properties.examples.methanol_ideal_VLE import config_dict
+
+
+# -----------------------------------------------------------------------------
+# Get default solver for testing
+solver = get_solver()
+
+
+def _as_quantity(x):
+    unit = pyunits.get_units(x)
+    if unit is None:
+        unit = pyunits.dimensionless
+    return value(x) * unit._get_pint_unit()
+
+
+# Test for configuration dictionaries with parameters from Properties of Gases
+# and liquids 4th edition
+class TestParamBlock(object):
```

Comment:
This test is not required - all of this gets covered by the tests in the modular properties code.